### DISCUSSION TITLE
CLI login failed

### DISCUSSION THREAD

**Discussion by Player1041** [deta/cloud-docs#313] [Help] [answered]
Tried to login using deta login on termux, didn't work

~ $ deta login
Please, log in from the web page. Waiting...
https://web.deta.sh/cli/33137
Error: Get "https://v1.deta.sh/spaces/": dial tcp: lookup v1.deta.sh on [::1]:53: read udp [::1]:52424->[::1]:53: read: connection refused
~ $

**Comment by abdelhai** [comment, marked as answer]
hi, use a deta access token https://docs.deta.sh/docs/cli/auth#deta-access-tokens

**Reply by dmnsea** [reply]
Simply use token changes nothing - tried both environment variable and `~/.deta/tokens` file

After that trying to simply `deta projects` brings same `Error: Get "https://v1.deta.sh/spaces/" ...` with trying to reach [::]:53

**Reply by dmnsea** [reply]
`[::1]:53`* (typo in prev message)

**Comment by Prog123-afk** [comment]
Do this:
`

> apt install proot-distro

> proot-distro install alpine

> proot-distro login alpine

>/data/data/com.termux/files/home/.deta/bin/deta login

`
